cilium/rockcraft.yaml

```diff
@@ -227,54 +241,36 @@ parts:
   bpftool:
     plugin: make
     source-type: git
-    source: https://git.kernel.org/pub/scm/linux/kernel/git/bpf/bpf-next.git
-    source-commit: "93270357daa949e4bed375b40d0a100ce04f3399"
+    source: https://github.com/libbpf/bpftool.git
+    source-tag: v7.0.0
```

Comment:
We used to pull `bpf-next` with a specific source commit that pointed to `v7.0.0` version of bpftool. The issue is `bpf-next` is huge and pulls a lot of kernel code just to build simple `bpftool`, the git server being slow there also does not help. 

So instead we are switching to use the official `bpftool` mirror that only syncs `bpftool` related header files and code, which is way smaller and builds way faster. 

This saves  ~`8GiB` of disk space.